src/resources/HttpResource.ts

```diff
@@ -66,6 +68,6 @@ export default class HttpResource<TApi: HttpApi = HttpApi> extends Resource<
   }
 
   getConnectionBase(path: string, args: Args) {
-    return this.api.getUnpaginatedConnection<Object>(path, args);
+    return this.api.getUnpaginatedConnection<Record<string, any>>(path, args);
```

Comment:
ditto, i don't know the diffference in behavior tho between flow's Object and `{}` in TS